orttraining/orttraining/core/optimizer/graph_transformer_utils.cc

```diff
@@ -63,6 +66,11 @@ std::vector<std::unique_ptr<GraphTransformer>> GeneratePreTrainingTransformers(T
       transformers.emplace_back(onnxruntime::make_unique<GeluFusion>(compatible_eps));
       transformers.emplace_back(onnxruntime::make_unique<LayerNormFusion>(compatible_eps));
       transformers.emplace_back(onnxruntime::make_unique<FastGeluFusion>(compatible_eps));
+
+      transformers.emplace_back(onnxruntime::make_unique<BiasGeluFusion>(compatible_eps));
+
+      transformers.emplace_back(onnxruntime::make_unique<GeluApproximation>(compatible_eps));
+
```

Comment:
Need to expose an session level config for this... 
For different transformer, we may want to set this differetnly